### PR TITLE
fix: homepage revision

### PR DESCRIPTION
**Pull request by ethangj**
Re-arranged the homepage for better information flow, updating CTAs, fixing some color issues and fixing the footer on Safari.

### CODE REVIEW

**Comment by ethangj** [comment]
I will redo this PR based on the theme change once that is implemented.

File: package-lock.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
       "version": "2.1.3",
       "resolved": "https://registry.npmjs.org/@nodelib/fs.scandir/-/fs.scandir-2.1.3.tgz",
       "integrity": "sha512-eGmwYQn3gxo4r7jdQnkrrN6bY478C3P+a/y72IJukF8LjB6ZHeB3c+Ehacj3sYeSmUXGlnA67/PmbM9CVwL7Dw==",
-      "dev": true,
```
Comment (hferentschik) [inline package-lock.json:11]: ooi, how did you install the `npm` dependencies? I am kind of intrigued that this lock file keeps changing on us. There must be different ways on how people install npm dependencies.

Comment (ethangj) [reply package-lock.json:11]: Just did npm install as per the Readme, weird.

**Comment by ethangj** [comment]
@hferentschik Fixed it being broken in the Safari (see screenshot.)

I like the left aligned but either way I don't see it as permanent, just temporary until @jsouth gets his hands in here.

Screen Shot 2020-04-28 at 8 31 55 AM

File: package-lock.json (hunk 1 above)
Comment (hferentschik) [reply package-lock.json:11]: not an issue really,  but I have seen this file changing forth and back now. Something must be different somewhere.

**Comment by hferentschik** [comment]
needs a rebase now

**Comment by ethangj** [comment]
@hferentschik could you resolve your change request on the footer?

**Comment by ethangj** [comment]
@hferentschik @sharepointoscar after many trials and tribulations, this is now good. please review and 👍

**Comment by ethangj** [comment]
@hferentschik updated with a new commit message

**Pull request merged**